src/main/java/org/jenkinsci/plugins/sonargerrit/gerrit/GerritReviewBuilder.java

```diff
@@ -47,6 +47,7 @@ public ReviewInput buildReview() {
     String reviewMessage = getReviewMessage(finalIssuesToComment);
     ReviewInput reviewInput = new ReviewInput().message(reviewMessage);
     reviewInput.tag = "autogenerated:sonar";
+    reviewInput.omitDuplicateComments = true;
```

Comment:
This should be configurable. By default, the current behaviour should be kept, i.e. `reviewInput.omitDuplicateComments = false`.

Also, this should be covered by automated tests.

Reply:
I've made it configurable by adding a checkbox to the ReviewConfig section.
omitDuplicateComments is just a flag that is set in the reviewInput. The Processing is done by gerrit.